scripts/deploy/index.js

```diff
@@ -0,0 +1,28 @@
+const { inputDir = './source' } = require('yargs').argv
+const program = require('commander')
+const deployS3 = require('../../lib/deploy/s3')
+const deployGhp = require('../../lib/deploy/ghp')
+
+const args = [
+  require.resolve('standard/bin/cmd'),
+  `${inputDir}/**/*.js`
+].concat(process.argv.slice(2))
+
+program
```

Comment:
Commander looks awesome, as discussed though can we use yargs here instead? It's already in the project. We can move it _all_ to commander at a later date :)